content/spin/v2/registry-tutorial.md

```diff
@@ -101,6 +101,16 @@ Lastly, let's run this Spin application:
 $ spin up -f ghcr.io/USERNAME/spin-react-fullstack:v1
 ```
 
+## Deploy a Spin App from GHCR
+
+You can deploy a Spin app from GHCR by passing the GHCR URL as a query parameter. To do so, just use the URL in this format: 
```

Comment:
Note that users can do the same directly from the cli, for any remote app reference, including from GHCR, docker.io, etc,  eg `spin deploy -f <remote-reference-here>`.

This brings the same disclaimer that you've added below re: needing a cloud account -- though in this case, the reference actually _can_ be private, as long as the user is authenticated locally to the registry in question, since the cli will pull it down prior to publishing it to Cloud's internal registry.

I wonder if supplying the CLI command here might be a more natural fit?  Or maybe in addition?

Reply:
Yeah that is the internal mechanism behind the Deploy to Cloud button and I don't think we want users using it.  If we want to offer direct deploy then we should add a UX for it (or modify `spin deploy` to use the `/deploy` endpoint instead of pushing to the Cloud registry).  In the meantime definitely change to `spin deploy -f <reference>`.